genie_registry/clinical.py

```diff
@@ -175,6 +175,33 @@ def _check_int_year_consistency(
     return ""
 
 
+def _check_year_death_validity(clinicaldf: pd.DataFrame) -> str:
+    """
+    Check if YEAR_DEATH >= YEAR_CONTACT
```

Comment:
Could you add more to the description here? Details about what is considered invalid vs valid in this scenario.